controllers/users.js

```diff
@@ -34,11 +34,11 @@ var error400 = function(msg, callback) {
   });
 };
 
-var getType = function(user, admins) {
+var getType = function(user) {
   if (user.roles && user.roles.length) {
     return user.roles[0];
   }
-  return admins[user.name] ? 'admin' : 'unknown';
```

Comment:
So again, this is probably controversial, but is sort of the keystone of this PR. Previously we have code-- either because we needed it or because we were being extra cautious-- which checked for CouchDB admins and injected the admin role into results. It's very unclear if we rely on this, and at least for LG admin users actually already have the `_admin` role.

@garethbowen we'll almost certainly have to talk about this. I'm very much getting the feeling that I'm pulling at some pretty long threads here.